lesson15/index.html

```diff
@@ -17,8 +17,16 @@
       <div class="modal-overlay" id="js-modal-overlay"></div>
       <div class="modal-content" id="js-modal-content">
         <div class="modal-body" id="js-modal-body">
-          <input type="number" name="" id="js-input-number" value="" class="input-number">
-          <button id="js-btn-fetch" class="btn-fetch">Get input num & Fetch Data</button>
+          <!-- <form class="form" id="js-form" onsubmit="init();"> -->
+          <form class="form" id="js-form">
+            <label for="js-input-number">数字
+              <input type="number" name="" id="js-input-number" value="" class="input-number">
```

Comment:
Why is the name empty?(also line26)

Reply:
If you don't add the name attribute, you won't be able to identify it when you send it to the server...
Thanks for pointing that out!
So I have fixed it that way.

fdaad37